.travis.yml

```diff
@@ -42,6 +42,15 @@ matrix:
     - env: RUN="unit"
     - env: RUN="unit-next" BOULDER_CONFIG_DIR="test/config-next"
     - env: RUN="coverage"
+    #
```

Comment:
Question: Can we not just stick a second version in the `go` section and use `TRAVIS_GO_VERSION` in `docker-compose.yml`? That way we don't need to keep adding/remove this big section of `env`s and just do a single line diff when changing/adding/removing versions.

Reply:
@rolandshoemaker I don't think this works (or I'm missing something). I tried this in https://github.com/letsencrypt/boulder/pull/3690/commits/a089de3b6a2e52e875b8ee29ffe52d82bdc8282c and it resulted in a build matrix like this:

![not what i wanted](https://user-images.githubusercontent.com/292650/39647142-ba71e536-4fab-11e8-8ccb-6ad24da143a6.png)

I think we'd need to do:
```
matrix:
  include:
    - go: 1.10.1
      env:  RUN="vet fmt migrations integration godep-restore errcheck generate dashlint rpm"   
    - go: 1.10.2
      env:  RUN="vet fmt migrations integration godep-restore errcheck generate dashlint rpm"                                                                                   
```
and that's right back to where we were duplication-wise with the added advantage of the `docker-compose.yml` for dev referencing an env var named `TRAVIS_GO_VERSION` outside of CI which seems more confusing than `GO_VERSION`.